avldb/build.sbt

```diff
@@ -0,0 +1,42 @@
+import sbt.Keys.testFrameworks
+
+name := "avldb"
```

Comment:
I did it in the same way as ergo-walet was integrated in ergo project.
I am not expert in Scala and especially SBT, so may be it can done in some other way.